.env.example

```diff
@@ -5,4 +5,5 @@ VITE_INFURA_SECRET=xxxxxx
 VITE_PLATFORM_ID=1
 VITE_LENS_URL = https://api.lens.dev/
 VITE_IPFS_BASE_URL=https://ipfs.io/ipfs/
-VITE_POH_SUBGRAPH_URL = https://api.thegraph.com/subgraphs/name/kleros/proof-of-humanity-mainnet
\ No newline at end of file
+VITE_POH_SUBGRAPH_URL = https://api.thegraph.com/subgraphs/name/kleros/proof-of-humanity-mainne
+VITE_SIGNATURE_AUTOTASK_URL=https://api.defender.openzeppelin.com/autotasks/xxxxxxxx-xxxx-xxx-xxxx-xxxxxxxxxxxxx/runs/webhook/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx/xxxxxxxxxxxxxxxxxxxxxxx
```

Comment:
could be more generic, VITE_SIGNATURE_API_URL

Reply:
and we can use the real url here to let anyone clone and have everything working